native/src/routes/SprungbrettOfferContainer.tsx

```diff
@@ -31,48 +17,14 @@ type SprungbrettOfferContainerProps = {
 }
 
 const SprungbrettOfferContainer = ({ route, navigation }: SprungbrettOfferContainerProps): ReactElement => {
-  const cities = useCities()
-  const [title, setTitle] = useState<string>('')
   const { cityCode, languageCode } = useCityAppContext()
   const alias = SPRUNGBRETT_OFFER_ROUTE
```

Comment:
(optinal) as this was not part of your PR, but saving this well named constant to a local variable does not really make sense.